GNUmakefile

```diff
@@ -66,11 +66,13 @@ GITLAB_TOKEN ?= ACCTEST1234567890123
 GITLAB_BASE_URL ?= http://127.0.0.1:8080/api/v4
 
 testacc-up: ## Launch a GitLab instance.
+	mkdir -p certs
+	openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes -keyout certs/example.key -out certs/example.crt -subj "/CN=example.com" -addext "subjectAltName=DNS:IP:127.0.0.1"
```

Comment:
Can we rename the certificate files (and probably also the CN) to something "more meaningful" ? It'snot really an example after all :)

Reply:
maybe also put them into a hidden or temporary folder?

Reply:
for the certs they are in a temporary folder and the .gitignore already has those listed so they are not committed.  I can rename the certs to something better

Reply:
changed it up a bit hopefully this is more of what you were thinking of

Reply:
> for the certs they are in a temporary folder and the .gitignore already has those listed so they are not committed.  I can rename the certs to something better

I was more thinking about a real "temporary" one (e.g. inside `$TEMPDIR`) but it's ok for a first merge :)